src/formatters.rs

```diff
@@ -0,0 +1,293 @@
+use anyhow::{anyhow, Result};
+use enum_dispatch::enum_dispatch;
+use geo::geometry::Geometry;
+use geojson;
+use polars::prelude::*;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use serde_json::Value;
+use std::io::Cursor;
+use std::io::Write;
+use wkb::geom_to_wkb;
+use wkt::TryFromWkt;
+
+/// Utility function to convert a polars series from WKT geometries to
+/// WKB geometries (as a string)
+fn convert_wkt_to_wkb_string(s: Series) -> PolarsResult<Option<Series>> {
+    let ca = s.str()?;
+    let wkb_series: Vec<Vec<u8>> = ca
+        .into_iter()
+        .map(|opt_wkt| {
+            opt_wkt
+                .map(|wkt_str| {
+                    let geom: Geometry<f64> =
+                        Geometry::try_from_wkt_str(wkt_str).expect("Failed to parse wkt");
+                    geom_to_wkb(&geom).expect("Failed to format geom")
+                })
+                .unwrap_or_else(Vec::new)
+        })
+        .collect();
```

Comment:
Perhaps to avoid the `.expect()`, could rewrite with the [`PolarsError::ComputeError`](https://docs.rs/polars/latest/polars/error/enum.PolarsError.html) variant or another one of the variants, e.g.:
```suggestion
   let wkb_series = ca
        .into_iter()
        .map(|opt_wkt| {
            opt_wkt
                .map(|wkt_str| {
                    Geometry::try_from_wkt_str(wkt_str)
                        .map_err(|err| {
                            PolarsError::ComputeError(
                                format!("Failed to parse wkt: {err:?}").into(),
                            )
                        })
                        .and_then(|geom: Geometry<f64>| {
                            geom_to_wkb(&geom).map_err(|_| {
                                PolarsError::ComputeError("Failed to format geom: {err:?}".into())
                            })
                        })
                })
                .unwrap_or_else(|| Ok(Vec::new()))
        })
        .collect::<Result<Vec<Vec<u8>>, _>>()?;
```

Reply:
This seems good to me. I opened #25 to think through the error handling strategy for the library in general. Started using anyhow for ease but I think we should go back at some point and implement proper errors using thiserror.